src/gei/GithubSecretScanningAlertServiceFactory.cs

```diff
@@ -0,0 +1,29 @@
+using Octoshift;
+using OctoshiftCLI.Contracts;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter;
+
+public sealed class GitHubSecretScanningAlertServiceFactory : ISecretScanningAlertServiceFactory
```

Comment:
We made a decision not to create namespaces unless it's absolutely necessary. We create ordinary classes and mark the public methods as `virtual` to be able to mock them.  
```suggestion
public class GitHubSecretScanningAlertServiceFactory
```